test/test_utils/sagemaker.py

```diff
@@ -227,6 +227,22 @@ def execute_local_tests(image, ec2_client):
         ec2_conn.run(f"$(aws ecr get-login --no-include-email --region {region})")
         ec2_conn.run(f"docker pull {image}")
         ec2_conn.run(f"tar -xzf {sm_tests_tar_name}")
+
+        stop_updates_command = """
+                    APT_DAILYS="apt-daily.service apt-daily-upgrade.service unattended-upgrades.service";
```

Comment:
why set bash env variables instead of string interpolation??

Reply:
Do you mean `APT_DAILYS`? The string `"apt-daily.service ..."` is repeated thrice in the command, so it makes sense to modularize it.